bot/exts/moderation/incidents.py

```diff
@@ -173,18 +265,33 @@ class Incidents(Cog):
         * If `Signal.ACTIONED` or `Signal.NOT_ACTIONED` were chosen, attempt to
           relay the incident message to #incidents-archive
         * If relay successful, delete original message
+        * Search the cache for the webhook message for this message, if found delete it.
```

Comment:
No other list item ends with a period.

Consider rewording to something like: "Delete quotation message if cached." Webhook message will make no sense to developers reading the docs without knowledge of the internals.

Reply:
Resolved in 88cefa6.